### ISSUE TITLE
installation of precompiled binaries triggers App Translocation on Sierra

### ISSUE THREAD

**Issue by cambecc** [yujitach/MenuMeters#41]
Download the precompiled binaries and install by double-clicking on `MenuMeters.prefPane`. This will trigger [App Translocation](http://lapcatsoftware.com/articles/app-translocation.html), which isolates `MenuMeters.app` to a read only filesystem with a random mount point:
```
$ df -h
Filesystem                                                                                        Size   Used  Avail Capacity iused      ifree %iused  Mounted on
/Users/cambecc/Library/PreferencePanes/MenuMeters.prefPane/Contents/Resources/MenuMetersApp.app  931Gi   40Gi  890Gi     5% 1330281 4293636998    0%   /private/var/folders/2z/9vfqf31x1l2bk7n92vrqz4500000gn/T/AppTranslocation/248A0569-0947-496E-9BD9-B66D59B5273E
```
The app seems to work but it's not ideal to have these weird file systems sitting around forever. One workaround is to strip the quarantine extended attribute from the .zip file before installation:
```
xattr -d com.apple.quarantine MenuMeters_1.9.3.zip
```
Is there a better way to resolve this for people using the precompiled binaries?

Most likely, building from source would not run into this issue.

**Comment by yujitach** [comment]
I knew this issue. I consider it an unintended behavior on the Apple's part: a standard .app in a .zip file, when moved from ~/Download to somewhere else using Finder, is cleared of this App Translocation behavior. Why doesn't the pref pane, when moved from ~/Download by double-clicking by System Preference, get the same behavior? I will file a bug report to Apple.

**Comment by lukehamburg** [comment]
Thanks @cambecc for pointing this out. For me, it wasn't enough to strip the quarantine attribute from the .zip file. I had to extract the prefPane bundle and then run the xattr with the recursive (-r) flag:

`xattr -dr com.apple.quarantine MenuMeters.prefPane/`

**Comment by onpaws** [comment]
As a data point, I only had to perform the one `xattr` that @cambecc recommended.

It can be a bit confusing because it's easy to run multiple instances of MenuMeters.

An exercise for someone with more time than me: [Preventing multiple app instances](http://stackoverflow.com/questions/684911/how-to-detect-whether-an-os-x-application-is-already-launched)

**Comment by yujitach** [comment]
@onpaws I guess it's not enough to prevent the newer copy to run if there is another instance already running. In the case of various versions of MenuMeters coexisting, we need old versions to kill itself...

**Comment by goudfroo** [comment]
Indeed, not ideal but one can deal with it. For example one can redefine 'df' by appending "grep -v MenuMeters" at the end of the command line.

**Comment by yujitach** [comment]
I consider this fixed in the new version 1.9.9, which runs outside of System Preferences. See https://github.com/yujitach/MenuMeters/releases/tag/1.9.9 .
